packages/frontend/components/ErrorToast.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+
+// TODO Remove this in future
+interface IErrorToast {
+  errorMessage: string;
+}
+
+const ErrorToast: FC<IErrorToast> = ({ errorMessage }) => {
```

Comment:
Created a basic custom Toast component, but should we use a library to handle the toasts?  Because this component does not disappear after some time nor it has a close button

Reply:
Let me think about this a little bit more. But I think we could implement something simple by ourselves without much trouble.

I'll do some testing, and If I see it gets too big... we might want to use a lib :P